pkg/termios/termios_test.go

```diff
@@ -267,3 +269,53 @@ func TestSet(t *testing.T) {
 		}
 	}
 }
+
+// This test tries to prevent people from breaking other operating systems.
+//
+// Compare:
+//
+//	GOOS=linux   GOARCH=amd64 go doc golang.org/x/sys/unix.Termios
+//	GOOS=darwin  GOARCH=amd64 go doc golang.org/x/sys/unix.Termios
+//	GOOS=openbsd GOARCH=amd64 go doc golang.org/x/sys/unix.Termios
+//
+// It's all a mess.
+//
+// This at least makes sure that the package compiles on all platforms.
+func TestCrossCompile(t *testing.T) {
```

Comment:
Can you also add 

testutil.SkipIfInVMTest(t)

inside our VM infra, the go tool is not available.

Reply:
Done. I thought that CIFuzz was failing even on my prior change for unrelated reasons, though, even from before I added a test. So I'd just ignored its failure this time.